recipes/newrelic/apm/php-agent/php.yml

```diff
@@ -71,10 +124,16 @@ install:
 
     restart_services:
       cmds:
+        - echo "Restarting the services as needed to pick up newrelic.ini information."
         - |
-          if [ -z "{{.NR_PHP_RESTART}}" ]; then
-            echo "Restarting php-fpm"
-            sudo service php-fpm restart
+          if [ "{{.NR_PHP_RESTART}}" = "y" ]  && [ -f "{{.TMP_INSTALL_DIR}}/processes_to_restart.txt" ]; then
```

Comment:
If non-silent installs, when will the user be prompted about the restart? (I ask because I want to us to include the process names that will be restarted in the associated prompt).

Reply:
The warning will occur [here](https://github.com/newrelic/open-install-library/blob/4bb4b4e41336a0b50f4e96c0be908cae20a29d8c/recipes/newrelic/apm/php-agent/php.yml#L138).  It could also use a big warning like [this](https://github.com/newrelic/open-install-library/blob/e7347282f607e079f528d8199b678f29dbeb4fcb/recipes/newrelic/apm/dotNet/linux-systemd.yml#L58) before we restart (in non-silent mode).

Todos for the recipe that:
1) Silent mode needs to be incorporated like [this](https://github.com/newrelic/open-install-library/blob/e7347282f607e079f528d8199b678f29dbeb4fcb/recipes/newrelic/apm/dotNet/linux-systemd.yml#L71).  This can either be it's own ticket so it is consistent across the tasks, or I can work on it here as the blueprint to be used in other tasks.
2) Add error exit codes.  This can either be it's own ticket so it is consistent across the tasks, or I can work on it here as the blueprint to be used in other tasks.

Reply:
Separate tickets sounds like a better option to me.